src/applications/personalization/profile/tests/actions/paymentInformation.unit.spec.js

```diff
@@ -594,12 +594,13 @@ describe('actions/paymentInformation', () => {
         });
 
         it('reports the correct data to Google Analytics', () => {
-          expect(recordEventSpy.firstCall.args[0]).to.deep.equal({
```

Comment:
This check always passes locally, but failed almost every time on CI. I have no idea why :(

Example http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/eph-dd4edu-ga/1/tests

<img width="922" alt="Screen Shot 2021-04-12 at 2 05 29 PM" src="https://user-images.githubusercontent.com/20728956/114462626-2a2f2880-9b98-11eb-9ec5-f889cbe25595.png">